butane/Cargo.toml

```diff
@@ -1,26 +1,26 @@
 [package]
-name = "butane"
-version = "0.5.1"
 authors = ["James Oakley <james@electronstudio.org>"]
-edition.workspace = true
+categories = ["database"]
 description = "An ORM with a focus on simplicity and on writing Rust, not SQL."
-readme = "../README.md"
+documentation = "https://docs.rs/butane/"
+edition.workspace = true
 keywords = ["database", "orm", "sql"]
-categories = ["database"]
 license = "MIT OR Apache-2.0"
+name = "butane"
+readme = "../README.md"
 repository = "https://github.com/Electron100/butane"
-documentation = "https://docs.rs/butane/"
+version = "0.5.1"
 
 [features]
-default = ["datetime", "json", "uuid"]
-json = ["butane_core/json"]
-sqlite = ["butane_core/sqlite"]
-sqlite-bundled = ["butane_core/sqlite-bundled"]
-pg = ["butane_core/pg"]
 datetime = ["butane_core/datetime", "butane_codegen/datetime"]
 debug = ["butane_core/debug"]
+default = ["datetime", "json", "uuid"]
```

Comment:
putting this in the middle is also a significant degradation IMO.  I suspect most Rust developers look at Cargo.toml of libraries they are using, and the default feature list is very commonly what they are looking for.

I've created a bunch of issues like this https://github.com/tbrand/cargo-tomlfmt/issues/25 , as that tool is more likely to support Cargo.toml specific formatting than the more general `taplo`

Reply:
More tools at https://github.com/TimonPost/cargo-toml-format#similar-work .  I'll try them out and see what works best.